libraries/ESP8266WiFi/src/ESP8266WiFiMulti.cpp

```diff
@@ -91,14 +91,16 @@ wl_status_t ESP8266WiFiMulti::run(void) {
                 for(auto entry : APlist) {
                     if(ssid_scan == entry.ssid) { // SSID match
                         known = true;
-                        if(rssi_scan > bestNetworkDb) { // best network
-                            if(sec_scan == ENC_TYPE_NONE || entry.passphrase) { // check for passphrase if not open wlan
-                                bestNetworkDb = rssi_scan;
-                                bestChannel = chan_scan;
-                                bestNetwork = entry;
-                                memcpy((void*) &bestBSSID, (void*) BSSID_scan, sizeof(bestBSSID));
+                        if (entry.priority <= bestNetwork.priority){

```

Comment:
Shouldn't it be the other way around? It seems that lower values of 'priority' mean higher priority. If that's intended, maybe give this variable some other name (like 'order'?)

Reply:
Maybe this is just my background but i think of it like "number one priority" or "your first priority is x, your second is y" etc.

DNS MX records work in a similar way, lower number = higher priority

Reply:
UNIX nice's the same way (1=important, 20=only if nothing else to do).  How many Arduino users are aware of this, though, I can't say

Reply:
This was my first pull request to an open source project. Now that I've changed the variable to order instead of priority do I just wait? Or should I close it and re- request?